script/HelperConfig.s.sol

```diff
@@ -6,22 +6,20 @@ import {Script} from "forge-std/Script.sol";
 import {MockV3Aggregator} from "../test/mocks/MockV3Aggregator.sol";
 import {ERC20Mock} from "@openzeppelin/contracts/mocks/ERC20Mock.sol";
 
-
 contract HelperConfig is Script {
     struct NetworkConfig {
-
         address wethUsdPriceFeed;
         address wbtcUsdPriceFeed;
         address weth;
         address wbtc;
         uint256 deployerKey;
     }
+
     uint8 public constant DECIMALS = 8;
     int256 public constant ETH_USD_PRICE = 2000e8;
     int256 public constant BTC_USD_PRICE = 1000e8;
     uint256 public constant DEFAULT_ANVIL_KEY = 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80;
```

Comment:
Is this definitely ok to commit given that its a private key?

Reply:
Yes, this is a fake private key given to users in foundry. To confirm, in foundry type " anvil" in your command line and it will provide you with private keys and public accounts to use for testing.